public/app/core/reducers/fn-slice.ts

```diff
@@ -1,56 +1,31 @@
-import { Theme as MuiTheme } from '@mui/material';
+import { Theme as MuiTheme, createTheme as muiCreateTheme } from '@mui/material';
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { WritableDraft } from 'immer/dist/internal';
 
-import { createTheme, GrafanaThemeType } from '@grafana/data';
+import { GrafanaThemeType } from '@grafana/data';
 
-import { GrafanaTheme } from '../../../../packages/grafana-data/src/types/theme';
 import { AnyObject } from '../../fn-app/types';
 
 export interface FnGlobalState {
   FNDashboard: boolean;
   uid: string;
   slug: string;
   mode: GrafanaThemeType.Light | GrafanaThemeType.Dark;
-  theme: FnTheme;
+  theme: MuiTheme;
```

Comment:
Why do We use MuiTheme?
I asked to use grafana theme

It will be easier to override colors.
Otherwise we will have to maintain MUiTheme and pay attention to grfana theme